client/src/app/components/sidebar/object-inpsector/object-insepctor.component.ts

```diff
@@ -0,0 +1,90 @@
+import {Component, Input, DoCheck, OnChanges} from "@angular/core";
+import {Validators, FormBuilder, FormGroup, REACTIVE_FORM_DIRECTIVES, FORM_DIRECTIVES} from "@angular/forms";
+import {ExpressionInputComponent} from "../../forms/inputs/types/expression-input.component";
+import * as _ from "lodash";
+import {BehaviorSubject} from "rxjs";
+import {InputProperty} from "../../../models/input-property.model";
+
+require ("./object-inpsector.component.scss");
+
+@Component({
+    selector: "object-inspector",
+    directives: [
+        ExpressionInputComponent,
+        REACTIVE_FORM_DIRECTIVES,
+        FORM_DIRECTIVES
+    ],
+    template: `
+            <form class="object-inspector-component">
+                <div class="formHead">
+                     <span class="edit-text">Edit</span>
+                    <i class="fa fa-info-circle info-icon"></i>
+                </div>
+            
+                <div class="form-group">
+                    <label for="inputId">ID</label>
+                    <input type="text" 
+                           name="inputId" 
+                           class="form-control"
+                           [(ngModel)]="selectedProperty.id">
+                </div>
+                
+                <div class="form-group">
+                    <label for="inputType">Type</label>
+                    
+                    <select class="form-control" 
+                    name="dataType"
+                    [(ngModel)]="selectedProperty.type" required>
+                        <option *ngFor="let propertyType of propertyTypes" [value]="propertyType">
+                            {{propertyType}}
+                        </option>
+                    </select>
+                </div>
+                
+                <div class="form-group">
+                    <label for="inputValue">Value</label>
+                    <expression-input [inputControl]="objectInspectorForm.controls['expression']"
+                                      [(expression)]="selectedProperty.value">
```

Comment:
Don't bind form controls and values separate for external assembly, pass the control with the value assigned and change it on the control itself if needed.